class-metamodel/class-and-attribute-subsystem/class-attribute.xmm

```diff
@@ -44,7 +44,7 @@ attributes
     Domain {I, R21, R23}
     To identifier {R21}
     Ref {I2, R23}
-    Rnum {I2, R23}
+    Rnum {I, I2, R23}
```

Comment:
for better explanation of this correction see line 113 in this file

Reply:
You are on to something here!  See my comment on R21